pkg/sippyserver/dbloader.go

```diff
@@ -334,6 +335,67 @@ func getOrCreateTestID(
 	return testID, nil
 }
 
+func variantsKey(variants []string) string {
+	v := make([]string, len(variants))
+	copy(v, variants)
+	sort.Strings(v)
+	return strings.Join(v, ",")
+}
+
+func appendJobIssuesFromVariants(jobCache map[string]*models.ProwJob, jobIssues map[string][]jira.Issue) error {
+	// variantSetMap maps a sorted names of variants to a set of the variants for easy comparison
+	variantsSetMap := map[string]sets.String{}
+	// variantsIssuesMap maps a sorted names of variants to a slice of issues
+	variantsIssuesMap := map[string][]jira.Issue{}
```

Comment:
I'm thinking this code needs to check the AffectsVersion on the Jira issue, and only match jobs in that release. I believe it's multi-valued so if the jira had multiple versions affected, we can tie them to all.